plugins/catalog-backend-module-scaffolder-entities/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-scaffolder-entities",
```

Comment:
Hmm, trying to think of a better name here, but I guess it's fine for a catch all if we decide to add some other things other than just the `kind: Template` stuff here too.

Reply:
Yeah we pondered it, started out with `-template-kind` but given what the name of the processor was, we kinda shifted to the more generic naming. Funnily though, the module is still named `catalogModuleTemplateKind`, and that's supposedly the future ... :) Should THAT one be renamed to `catalogModuleScaffolderEntityModel` or something like that?